shared/js/background/chrome-events.es6.js

```diff
@@ -222,6 +241,8 @@ let onStartup = () => {
             .catch(e => console.log(e))
 
         https.sendHttpsUpgradeTotals()
+
+        Companies.buildFromStorage()
```

Comment:
This forces the companies module to build its contents from browser memory when the extension starts, rather than when the module gets loaded. It brings the companies module slightly more in line with other modules that depend on storage.

This was necessary so the Tab class can be used in an extension environment.

To test:

1. Clean top trackers data
2. Visit a site 30 times to top trackers data is restored from scratch
3. Restart the browser
4. The top trackers data should be intact